ts/import-csv/ImportCsvPage.svelte

```diff
@@ -56,17 +57,25 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         deckColumn,
         guidColumn,
     );
-    $: getCsvMetadata(path, delimiter, undefined, isHtml).then((meta) => {
+    $: getCsvMetadata(path, delimiter, undefined, undefined, isHtml).then((meta) => {
         columnLabels = meta.columnLabels;
         preview = meta.preview;
     });
-    $: if (globalNotetype?.id !== lastNotetypeId || delimiter !== lastDelimeter) {
+    $: if (
+        globalNotetype?.id !== lastNotetypeId ||
+        lastDeckId !== deckId ||
```

Comment:
Is it necessary to rerun `getCsvMetadata()` after the deck has changed? I think it doesn't have any influence on other settings or the preview, does it?

Reply:
I think you're right; I incorrectly assumed that the deck in the returned metadata would be used when importing the file, so I thought it needed to be updated.